### PR TITLE
Add more specific administrator language

### PR DESCRIPTION
**Pull request by keylimesoda**
There are many administrator/admin roles in Entra.  This update clarifies the specific/correct admin role (Global Administrator).

**Instructions:** _Add any supporting information, such as a description of the PR changes, here._

---
> [!NOTE]
> The following guidance is for Microsoft employees only. Community contributors can ignore this message; our content team will manage the status.
<details><summary><i>After you've created your PR</i>, expand this section for tips and additional instructions.</summary>

- **do not merge** is the default PR status and is automatically added to all open PRs that don't have the **ready to merge** label.
- Add the **ready for content review** label to start a review. Your PR won't be reviewed until you add this label.
- If your content reviewer requests changes, review the feedback and address accordingly as soon as possible to keep your pull request moving forward. After you address the feedback, remove the **changes requested** label, add the **review feedback addressed** label, and select the **Re-request review** icon next to the content reviewer's alias. If you can't add labels, add a comment with `#feedback-addressed` to the pull request.
- After the content review is complete, your reviewer will add the **content review complete** label. When the updates in this PR are ready for external customers to use, replace the **do not merge** label with **ready to merge** and the PR will be merged within 24 working hours.
- Pull requests that are inactive for more than 6 weeks will be automatically closed. Before that, you receive reminders at 2 weeks, 4 weeks, and 6 weeks. If you still need the PR, you can reopen or recreate the request.

For more information, see the [Content review process summary](https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/614263/Content-workflow).

</details>

### CODE REVIEW

**Review by Lauragra — APPROVED** [review]

File: api-reference/beta/api/subscription-list.md [hunk 1]
```diff
@@ -98,13 +98,13 @@ Most commonly, an application wants to retrieve subscriptions that it originally
...
 For such scenarios, a delegated permission Subscription.Read.All is required.
 
 | Context of the calling app | Response contains |
 |:-----|:---------------- |
 | App is calling on behalf of the signed-in user (delegated permission). *The user is a non-admin*. <br/>-and-<br/>App has the permission Subscription.Read.All<br/><br/>Note: This scenario applies to both personal Microsoft accounts and work/school accounts. | Subscriptions created by **any app** for the signed-in user only. |
-| App is calling on behalf of the signed-in user (delegated permission). *The user is an admin*.<br/>-and-<br/>App has the permission Subscription.Read.All<br/><br/>Note: This scenario only applies to work/school accounts. | Subscriptions created by **any app** for **any user** in the directory.|
+| App is calling on behalf of the signed-in user (delegated permission). *The user is a Global Administrator*.<br/>-and-<br/>App has the permission Subscription.Read.All<br/><br/>Note: This scenario only applies to work/school accounts. | Subscriptions created by **any app** for **any user** in the directory.|
```
Comment (FaithOmbongi) [inline api-reference/beta/api/subscription-list.md:107]: Can we use a lesser-privileged role?
```suggestion
| App is calling on behalf of the signed-in user (delegated permission). *The user is an authorized administrator, such as a Global Reader*.<br/>-and-<br/>App has the permission Subscription.Read.All<br/><br/>Note: This scenario only applies to work or school accounts. | Subscriptions created by **any app** for **any user** in the directory.|
```

File: api-reference/v1.0/api/subscription-list.md
Comment (FaithOmbongi) [inline api-reference/v1.0/api/subscription-list.md:1]: Same feedback as the beta doc

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Hi @keylimesoda - I've suggested some changes in line with the org initiative to burndown Global Admin dependencies and references.

File: api-reference/v1.0/api/subscription-list.md
Comment (keylimesoda) [reply api-reference/v1.0/api/subscription-list.md:1]: My understanding from the developer is that this is the correct permission.

There is a valuable secondary conversation as to if we need to update the code to allow for a lesser permissions.

**Review by keylimesoda — COMMENTED** [review]
The permission here is correct according to the dev who owns this code.

We may want to look at a code change in the future that reduces this permissions.

**Pull request merged**